### PR TITLE
(BIDS-3186) summary chart v2.5

### PR DESCRIPTION
**Pull request by LuccaBitfly**
This PR does 2 things:
1. Add the new premium perks for the chart.
image
3. Implement the v2.5 version of the summary chart
image

### CODE REVIEW

**Review by MauserBitfly — APPROVED** [review]
LGTM

File: backend/pkg/api/data_access/user.go [hunk 1]
```diff
@@ -457,7 +481,7 @@ func (d *DataAccessService) GetProductSummary() (*t.ProductSummary, error) {
 				PricePerMonthEur:         74.99,
 				PricePerYearEur:          719.88,
 				ProductIdMonthly:         "vdb_addon_1k",
-				ProductIdYearly:          "vdb_addon_1k.yearly",
+				ProductIdYearly:          "vdb_addon_1k.year	ly",
```
Comment (remoterami) [inline backend/pkg/api/data_access/user.go:484]: typo
```suggestion
				ProductIdYearly:          "vdb_addon_1k.yearly",
```

**Review by remoterami — CHANGES_REQUESTED** [review]
some minor issues, ptal

**Comment by remoterami** [comment]
lgtm

**Review by remoterami — APPROVED** [review]

**Pull request merged**